awkward-cpp/include/awkward/common.h

```diff
@@ -91,22 +89,6 @@ extern "C" {
         out.filename = filename;
         out.identity = identity;
         out.attempt = attempt;
-        out.pass_through = false;
-        return out;
-    };
-
-  inline struct Error
-    failure_pass_through(
-      const char* str,
-      int64_t identity,
-      int64_t attempt,
-      const char* filename) {
-        struct Error out;
-        out.str = str;
-        out.filename = filename;
-        out.identity = identity;
-        out.attempt = attempt;
-        out.pass_through = true;
```

Comment:
I'm removing this because, seeing as none of our kernels use it, it's unlikely to be useful. I assume it's legacy from the C++ era?

Reply:
Yes, that's right. It looks like we're letting `pass_through` be undefined now, which is a little nerve-wracking. If we're going to change the `Error` struct, we'll have to carefully check that nothing relies on its exact memory layout (in the Numba code). Meanwhile, if you want to set it to always `false`, just so that it's not undefined, that sounds safer.